src/server.c

```diff
@@ -5554,6 +5554,13 @@ int processCommand(client *c) {
         c->cmd->proc != quitCommand &&
         c->cmd->proc != resetCommand)
     {
+        if (c->cmd->proc == subscribeCommand ||
+            c->cmd->proc == psubscribeCommand)
+        {
+            rejectCommandFormat(c, "%s inside MULTI is not allowed",
+                tolower(c->cmd->name[0]) == 'p' ? "PSUBSCRIBE" : "SUBSCRIBE");
```

Comment:
We print lowercase commands around, so let's just stick with:
```suggestion
               c->cmd->name);
```

There are more subscribe commands coming, which will make this ugly.

Reply:
:+1: updated, thanks!